definitions.yml

```diff
@@ -282,7 +282,24 @@ definitions:
       name:
         type: string
       logo:
-        type: string
+        type: object
```

Comment:
We cannot do it like that, since logo is a required property. Existing decoders will fail due to the changed structure